crates/holochain/src/core/queue_consumer/sys_validation_consumer.rs

```diff
@@ -32,23 +32,17 @@ pub fn spawn_sys_validation_consumer(
         conductor.task_manager(),
         (tx.clone(), rx),
         move || {
-            if let Some(representative_agent) =
-                get_representative_agent(&conductor, &network.dna_hash())
-            {
-                Either::Left(sys_validation_workflow(
-                    workspace.clone(),
-                    current_validation_dependencies.clone(),
-                    trigger_app_validation.clone(),
-                    trigger_publish.clone(),
-                    trigger_self.clone(),
-                    network.clone(),
-                    keystore.clone(),
-                    representative_agent,
-                ))
-            } else {
-                tracing::warn!("No agent found for DNA, skipping sys validation");
```

Comment:
I actually just saw exactly this warning in my logs in Moss. It seems that there is a time window before the conductor is ready where the sys validation workflow already runs but `get_representative_agent()` returns `None`. After the line `Conductor ready.` in the logs the warnings stop.